benchmarks/django_simple/scenario.py

```diff
@@ -3,8 +3,8 @@
 
 
 class DjangoSimple(bm.Scenario):
-    tracer_enabled = bm.var(type=bool)
-    profiler_enabled = bm.var(type=bool)
+    tracer_enabled = bm.var(converter=bm.to_bool)
```

Comment:
wonder if we should just expose `bm.bool()` ?

```suggestion
    tracer_enabled = bm.bool()
```

just a random thought/suggestion.

Reply:
Hi Brett! 😃 Thank you for your review and the comment!
Totally agree with you, this call looks more readable. I will update the code then :)